app/auth/login.tsx

```diff
@@ -52,15 +52,15 @@ export const Login = ({ errorMessage }: { errorMessage: string }) => {
           ) : null}
           <Flex gap="2" direction="column" align="center">
             <Form action="/auth/github" method="post">
-              <Button type="submit">
+              <Button type="submit" disabled={!loaderData.githubEnabled}>
                 <Flex gap="1">
                   <GithubIcon width="16" />
                   Login with GitHub
                 </Flex>
               </Button>
             </Form>
             <Form action="/auth/google" method="post">
-              <Button type="submit" disabled>
+              <Button type="submit" disabled={!loaderData.googleEnabled}>
```

Comment:
`!l` and`!i` is hard to read, please use 
```suggestion
              <Button type="submit" disabled={loaderData.isGoogleEnabled === false}>
```

Reply:
Fixed